tests/bundle/test_storage_rules.py

```diff
@@ -336,6 +348,27 @@ def build_userop_for_factory(w3, entrypoint_contract, factory_contract, rule):
     StorageTestCase(
         "external_storage", STAKED, SENDER, build_userop_for_sender, assert_error
     ),
+    StorageTestCase(
+        "entryPoint_call_balanceOf",
+        UNSTAKED,
+        SENDER,
+        build_userop_for_sender,
+        assert_error,
+    ),
+    StorageTestCase(
+        "eth_value_transfer",
+        UNSTAKED,
+        SENDER,
+        with_ether_value(build_userop_for_sender),
+        assert_error,
+    ),
+    StorageTestCase(
```

Comment:
must make sure we test account deposit with both `entrypoint.call{value}` and `entrypoint.depositTo{value}()`
(one of them is the default account impl, the other should have explicit test

Reply:
Added.